mindsdb/integrations/handlers/auto_ts_handler/auto_ts_handler.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+from auto_ts import auto_timeseries as ATS
+from typing import Optional
+import dill
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+
+class Auto_ts_Handler(BaseMLEngine):
+    """
+    Auto_ts handler class
+    """
+    name = 'auto_ts'
+
+    @staticmethod
+    def create_validation(self, args):
```

Comment:
As I look at other handlers code, interface of this function should be:
```python
def create_validation(target, args=None, **kwargs):
```
It started to receive new param handler_storage BTW